BookingsAPI/Bookings.DAL/Commands/RemoveHearingCommand.cs

```diff
@@ -29,14 +31,26 @@ public async Task Handle(RemoveHearingCommand command)
         {
             var hearing = await _context.VideoHearings
                 .SingleOrDefaultAsync(x => x.Id == command.HearingId);
-            
+
             if (hearing == null)
             {
                 throw new HearingNotFoundException(command.HearingId);
             }
 
+            hearing.Participants.ToList().ForEach(x => RemoveQuestionnaires(x.Questionnaire));
```

Comment:
Let's have a chat about this. I'm thinking this is one of three options we've got. The other two being cascading deletes and raising events.

Reply:
It's changed